src/main/java/org/gradle/profiler/mutations/support/ScenarioSupport.java

```diff
@@ -0,0 +1,33 @@
+package org.gradle.profiler.mutations.support;
+
+import com.typesafe.config.Config;
+import org.gradle.profiler.ConfigUtil;
+
+import java.io.File;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class ScenarioSupport {
+
+    public static List<File> sourceFiles(Config config, String scenarioName, File projectDir, String key) {
+        return ConfigUtil.strings(config, key)
+            .stream()
+            .map(fileName -> openFile(fileName, projectDir, scenarioName))
+            .filter(Objects::nonNull)
+            .collect(Collectors.toList());
+    }
+
+    private static File openFile(String fileName, File projectDir, String scenarioName) {
```

Comment:
This doesn't really open files, but instead converts a `String` to a `File` while making sure it exists.

Reply:
removed